framework/include/constraints/EqualValueEmbededConstraint.h

```diff
@@ -0,0 +1,101 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#ifndef EqualValueEmbededConstraint_H
+#define EqualValueEmbededConstraint_H
```

Comment:
Follow the MOOSE convention of making these all caps: (EQUALVALUEEMBEDDEDCONSTRAINT_H)